### PR TITLE
Fixes #2403:  Resolve ktlint max line in player test package

### LINKED ISSUE #2403: Resolve ktlint max line in player test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length. 

Where to start?
https://github.com/oppia/oppia-android/blob/d2d55013605d318965594932b9224578cbc9a586/app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt#L291

### PR DESCRIPTION
**Pull request by som23ya**
Fixes #2403 : Resolve ktlint max line in player test package 

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt [hunk 1]
```diff
@@ -287,7 +287,7 @@ class ExplorationActivityTest {
 
   // TODO (#1855): Resolve ktlint max line in app module test
   @Test
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt:289]: You also need to remove these TODOs.

**Review by prayutsu — CHANGES_REQUESTED** [review]
@som23ya Please fix the title of the PR to follow our PR guidelines, it should match the format `Fix #issueNumber: Title of the PR`.
Also please add the `Fixes #issueNumber` keyword in the PR explanation to link the corresponding issue.

**Comment by som23ya** [comment]
> @som23ya Please fix the title of the PR to follow our PR guidelines, it should match the format `Fix #issueNumber: Title of the PR`.
> Also please add the `Fixes #issueNumber` keyword in the PR explanation to link the corresponding issue.

Thanks for helping on that, it had slipped my mind. :D

File: app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt [hunk 2]
```diff
@@ -347,9 +347,9 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
-  // TODO (#1855): Resolve ktlint max line in app module test
+  
   @Test
-  fun testAudioWithCellular_openRatioExploration_clickAudioIcon_changeConfiguration_checkOpensCellularAudioDialog() { // ktlint-disable max-line-length
+  fun testAudWithCell_openRatioExp_clickAudIcon_changeConfig_checkOpensCellAudDialog() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt:352]: this is doesn't sounds good `testAudWithCell`

`testAudioWithCellular_ratioExp_audioIcon_configChange_opensCellAudioDialog`
Let me know if this works, and also keep consistency with `configChange` everywhere.

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt:352]: follow a similar pattern with others as well.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @som23ya 
left a comment

**Comment by anandwana001** [comment]
please complete all the installations steps:
https://github.com/oppia/oppia-android/wiki#installation

ktlint Guide for any reference - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide

**Comment by anandwana001** [comment]
any update on this @som23ya ?

**Comment by anandwana001** [comment]
Closing this PR as no further update.

**Pull request closed without merging**